mmv1/third_party/terraform/resources/resource_storage_object_acl.go

```diff
@@ -227,7 +233,10 @@ func resourceStorageObjectAclUpdate(d *schema.ResourceData, meta interface{}) er
 			return fmt.Errorf("Error reading object %s in %s: %v", object, bucket, err)
 		}
 
-		objectOwner := sObject.Owner.Entity
+		var objectOwner string
+		if sObject.Owner != nil {
+			objectOwner = sObject.Owner.Entity
```

Comment:
Does calling `getRoleEntityChange` below with an empty objectOwner not cause any issues?

I guess if that's true this should be fine

Reply:
It only checks for equality in there (`if res.Entity == owner`), so should be good, I think.